CONCEPTS.md

```diff
@@ -0,0 +1,62 @@
+# gRPC Concepts Overview
+
+Remote Procedure Calls (RPCs) provide a useful abstraction for building
+distributed applications and services. The libraries in this repository
+provide a concrete implementation of the gRPC protocol, layered over HTTP/2.
+These libraries enable communication between clients and servers using any
+combination of the supported languages.
+
+
+## Interface
+
+Developers using gRPC start with the description of an RPC service (a collection
+of methods), and generate client and server side interfaces. The server implements
```

Comment:
... methods). _From this description, gRPC will generate client and server side interfaces in any of our supported languages_. The server...